rocketpy/Rocket.py

```diff
@@ -139,12 +139,12 @@ def __init__(
             in meters. Generally negative, meaning a negative position in the
             z axis which has an origin in the rocket's center of mass (without
             propellant) and points towards the nose cone.
-        distanceRocketPropellant : int, float
+        distanceRocketMotorReference : int, float
```

Comment:
I would recommend still using `distanceRocketPropellant`name instead of `distanceRocketMotorReference`, this way we garantee that we get the following error while runing the Flight class: `AttributeError: 'Rocket' object has no attribute 'distanceRocketPropellant'`

Otherwise we need to adjust some lines on the Flight.py file.

Reply:
To have backwards compatibility, could we have both arguments? Or would that be just a terrible code practice?